props/saber_fett263_buttons.h

```diff
@@ -104,20 +248,53 @@ SaberFett263Buttons() : PropBase() {}
         SaberBase::SetLockup(SaberBase::LOCKUP_NONE);
         auto_melt_on_ = false;
       }
+
+      // EVENT_PUSH
+      if (fabs(mss.x) < 3.0 && 
+          mss.y * mss.y + mss.z * mss.z > 120 &&
+          fusor.gyro().len2() < 1000 &&
+          fusor.swing_speed() < 30 &&
+          fusor.gyro().x < 10) {    
+        Event(BUTTON_NONE, EVENT_PUSH);
+      }
+
     } else {
-      // Swing On gesture control this portion allows fine tuning of speed needed to ignite
+      // EVENT_SWING - Swing On gesture control to allow fine tuning of speed needed to ignite
       if (millis() - saber_off_time_ < MOTION_TIMEOUT) {
         SaberBase::RequestMotion();
-        // Edit '250' value in line below to change swing on sensitivity when off
-        // 250 ~ 400 work best in testing
-        if (!swinging_ && fusor.swing_speed() > 250) {
+        if (!swinging_ && fusor.swing_speed() > FETT263_SWING_ON_SPEED) {
```

Comment:
This will really only work if FETT263_SWING_ON_SPEED is less than 250.
If it is higher, DetectSwing() will emit a swing event anyways.

Reply:
I forgot you added DetectSwing(), I moved to if(SaberBase::IsOn) to allow it to function properly, there are users who want to increase this value based on their hilt.